pages/curriculum.tsx

```diff
@@ -13,15 +13,20 @@ const Curriculum = () => {
 
   if (data) {
     const { lessons } = data
+    console.log(lessons)
+
     return (
       <Layout>
-        <Card title="Curriculum">
-          <ul>
-            {lessons.map((e: any) => (
-              <li key={e.id}>{e.title}</li>
-            ))}
-          </ul>
-        </Card>
+        {lessons
+          .sort((a: any, b: any) => a.order - b.order)
```

Comment:
Don't use `any` type when you know the type of the item. This defeats the purpose of having TypeScript. 

For this case you should be declaring the type of lessons like

`type Lesson = {
  order: number
   title: string
    ...
}`